pkgs/by-name/gh/ghstack/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "ghstack";
+  version = "v0.9.3";
```

Comment:
```suggestion
  version = "0.9.3";
```

Reply:
Thank you for the suggestion. I've updated my commit.